src/test_utils/index.tsx

```diff
@@ -75,6 +74,18 @@ const renderWithRouterMatch = (
   };
 };
 
+// Bind custom query methods to screen
+// https://github.com/testing-library/dom-testing-library/issues/516
+const boundQueries = Object.entries(customQueries).reduce(
+  (q, [queryName, queryFn]) => {
+    // eslint-disable-next-line no-param-reassign
+    q[queryName] = queryFn.bind(null, document.body);
+    return q;
+  },
+  {}
+) as Screen<CustomQueriesType>;
```

Comment:
What's the reason to type this as `Screen` rather than `BoundFunctions` like in the GitHub issue?